packages/oseq/oseq.0.3/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+synopsis:
+  "Simple list of suspensions, as a composable lazy iterator that behaves like a value"
+description:
+  "Extends the new standard library's `Seq` module with many useful combinators."
+maintainer: "simon.cruanes.2007@m4x.org"
+authors: "Simon Cruanes"
+license: "BSD-2-clauses"
+tags: ["sequence" "iterator" "seq" "pure" "list"]
+homepage: "https://github.com/c-cube/oseq/"
+doc: "https://c-cube.github.io/oseq/"
+bug-reports: "https://github.com/c-cube/oseq/issues"
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.0"}
```

Reply:
The purpose of this change is to remove the `{build}` tag and signal the minimal dune version to be a known one. The second one is optional but too many packages just have dune in their dependencies without considering the minimal version required, it's always good to know that it has been checked before.